pkg/cmd/server/api/helpers.go

```diff
@@ -12,6 +13,82 @@ import (
 	"github.com/openshift/origin/pkg/cmd/server/crypto"
 )
 
+func GetMasterFileReferences(config *MasterConfig) []*string {
+	refs := []*string{}
+
+	refs = append(refs, &config.ServingInfo.ServerCert.CertFile)
+	refs = append(refs, &config.ServingInfo.ServerCert.KeyFile)
+	refs = append(refs, &config.ServingInfo.ClientCA)
+
+	refs = append(refs, &config.EtcdClientInfo.ClientCert.CertFile)
+	refs = append(refs, &config.EtcdClientInfo.ClientCert.KeyFile)
+	refs = append(refs, &config.EtcdClientInfo.CA)
+
+	if config.EtcdConfig != nil {
+		refs = append(refs, &config.EtcdConfig.ServingInfo.ServerCert.CertFile)
+		refs = append(refs, &config.EtcdConfig.ServingInfo.ServerCert.KeyFile)
+		refs = append(refs, &config.EtcdConfig.ServingInfo.ClientCA)
+		refs = append(refs, &config.EtcdConfig.StorageDir)
+	}
+
+	if config.OAuthConfig != nil {
+		refs = append(refs, &config.OAuthConfig.ProxyCA)
+	}
+
+	if config.AssetConfig != nil {
+		refs = append(refs, &config.AssetConfig.ServingInfo.ServerCert.CertFile)
+		refs = append(refs, &config.AssetConfig.ServingInfo.ServerCert.KeyFile)
+		refs = append(refs, &config.AssetConfig.ServingInfo.ClientCA)
+	}
+
+	refs = append(refs, &config.MasterClients.DeployerKubeConfig)
+	refs = append(refs, &config.MasterClients.OpenShiftLoopbackKubeConfig)
+	refs = append(refs, &config.MasterClients.KubernetesKubeConfig)
+
+	return refs
+}
+
+func GetNodeFileReferences(config *NodeConfig) []*string {
+	refs := []*string{}
+
+	refs = append(refs, &config.ServingInfo.ServerCert.CertFile)
+	refs = append(refs, &config.ServingInfo.ServerCert.KeyFile)
+	refs = append(refs, &config.ServingInfo.ClientCA)
+
+	refs = append(refs, &config.MasterKubeConfig)
+
+	refs = append(refs, &config.VolumeDirectory)
+
+	return refs
+}
+
+func ResolvePaths(refs []*string, base string) error {
```

Comment:
This and relativize paths are both generic, so moving them to cmdutil makes sense.  We've already used similar logic in .kubeconfig file processing so we could collapse further into kube at some point.